lib/doorkeeper/oauth/authorization_code_request.rb

```diff
@@ -55,14 +54,6 @@ def pkce_supported?
         Doorkeeper.config.access_grant_model.pkce_supported?
       end
 
-      def validate_pkce_support
-        @invalid_request_reason = :not_support_pkce if grant &&
```

Comment:
Because `not_support_pkce` is used only here, so should remove from other files, too. (en.yml, invalid_request_response.rb)

Reply:
@linhdangduy thanks, missed those and removed them now :+1: